packages/components/src/Chip/Chip.tsx

```diff
@@ -44,17 +48,24 @@ export const Chip = ({
 
   return (
     <Tooltip message={tooltipMessage}>
-      <button
+      <Tag
         className={classes}
-        onClick={(ev: React.MouseEvent<HTMLButtonElement>) =>
-          onClick && onClick(value, ev)
+        onClick={(ev: React.MouseEvent<HTMLButtonElement | HTMLDivElement>) =>
+          onClick &&
+          !actAsFormElement &&
+          onClick(value || label, ev as React.MouseEvent<HTMLButtonElement>)
         }
         tabIndex={tabIndex}
-        onKeyDown={onKeyDown}
+        onKeyDown={(
+          ev: React.KeyboardEvent<HTMLButtonElement | HTMLDivElement>,
```

Comment:
Cool! I learned something new today - didn't realize you could just use `ev`.  Everywhere else we are using `event` so I'm not sure if there is a broader preference but I think this is neat & I support it!

Reply:
I updated it to be `event` to match the others I don't like standing out from the crowd! 

Really it's just a variable, we can call it anything. I vote for `keyboardEventTaylorsVersion` :)

Reply:
lol! Sounds good to me 😆